unit-10/homework/index.js

```diff
@@ -0,0 +1,49 @@
+// Exercise-1 In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả.
+const calcMultipliers = num => {
+  for (let i = 1; i < 11; i++) {
+    console.log (num * i)
+    }
+}
+
+//Exercise-2 Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6
+const getEvenNumber = num => {
+  if (1 <= num && num <= 30) return;
```

Comment:
Sai, nếu số nhập vào không nằm trong đoạn [1, 30] thì mới dừng
Nghĩa là hoặc bé hơn 1, hoặc hớn lơn 30
```suggestion
  if (1 < num || num > 30) return;
```